### PR TITLE
Removing redundant secrets from Admin and NCFS  Clusters

### PR DESCRIPTION
**Pull request by BigPeeg**

### CODE REVIEW

File: scripts/k8s_scripts/uk-south/create-ns-docker-secret-uks.sh [hunk 1]
```diff
@@ -11,8 +11,6 @@ FILESHARE_ACCOUNT_NAME=$(az keyvault secret show --name file-share-account --vau
 FILESHARE_KEY=$(az keyvault secret show --name file-share-key --vault-name gw-tfstate-Vault --query value -o tsv)
 TOKEN_USERNAME=$(az keyvault secret show --name token-username --vault-name gw-tfstate-Vault --query value -o tsv)
 TOKEN_PASSWORD=$(az keyvault secret show --name token-password --vault-name gw-tfstate-Vault --query value -o tsv)
-TRANSACTION_CSV=$(az storage account show-connection-string -g gw-icap-tfstate -n tfstate263 --query connectionString | tr -d '"')
-POLICY_CSV=$(az keyvault secret show --name policy-csv --vault-name gw-tfstate-Vault --query value -o tsv)
```
Comment (mpigram) [inline scripts/k8s_scripts/uk-south/create-ns-docker-secret-uks.sh:15]: So both of these have been backed into the config now?

File: scripts/k8s_scripts/uk-south/create-ns-docker-secret-uks.sh [hunk 2]
```diff
@@ -45,26 +43,26 @@ kubectl create -n $NAMESPACE01 secret generic ncfspolicyupdateservicesecret --fr
...
+kubectl create -n $NAMESPACE03 secret generic policyupdateserviceref --from-literal=TokenUsername=$TOKEN_USERNAME --from-literal=TokenPassword=$TOKEN_PASSWORD 
 
-kubectl create -n $NAMESPACE03 secret generic ncfspolicyupdateservicesecret --from-literal=username=$TOKEN_USERNAME --from-literal=password=$TOKEN_PASSWORD --from-literal=NcfsPolicyUpdateServiceEndpointCsv=$NCFS_POLICY_REF
+kubectl create -n $NAMESPACE03 secret generic ncfspolicyupdateservicesecret --from-literal=username=$TOKEN_USERNAME --from-literal=password=$TOKEN_PASSWORD
 
-kubectl create -n $NAMESPACE03 secret generic ncfspolicyupdateserviceref --from-literal=username=$TOKEN_USERNAME --from-literal=password=$TOKEN_PASSWORD --from-literal=NcfsPolicyUpdateServiceEndpointCsv=$NCFS_POLICY_REF
+kubectl create -n $NAMESPACE03 secret generic ncfspolicyupdateserviceref --from-literal=username=$TOKEN_USERNAME --from-literal=password=$TOKEN_PASSWORD
```
Comment (mpigram) [inline scripts/k8s_scripts/uk-south/create-ns-docker-secret-uks.sh]: Carrying on from my last comment, this now means we only need to supply to the token_username and password to these secrets now?

**Review by mpigram — APPROVED** [review]

File: scripts/k8s_scripts/uk-south/create-ns-docker-secret-uks.sh (hunk 1 above)
Comment (BigPeeg) [reply scripts/k8s_scripts/uk-south/create-ns-docker-secret-uks.sh:15]: Yes, the endpoints are now loaded in the value.yaml 
image

File: scripts/k8s_scripts/uk-south/create-ns-docker-secret-uks.sh (hunk 2 above)
Comment (BigPeeg) [reply scripts/k8s_scripts/uk-south/create-ns-docker-secret-uks.sh]: yes, just the credentials are required

**Pull request merged**